tests/test_data.py

```diff
@@ -643,7 +646,7 @@ def test_get_task_data(self):
 
         # Plot cycle count
         ts_data, samplerate = get_task_data(write_dir, self.subject, self.te_id, self.date, 'cycle')
-        self.assertEqual(len(ts_data), 7031)
+        self.assertEqual(len(ts_data), 7986)
```

Comment:
can you update this figure in the docs_dir (get_cycle_data.png) that was affected by changing the test data?